src/platform/tls_openssl.c

```diff
@@ -674,6 +696,9 @@ QuicTlsSecConfigCreate(
 
         SSL_CTX_set_max_early_data(SecurityConfig->SSLCtx, UINT32_MAX);
         SSL_CTX_set_client_hello_cb(SecurityConfig->SSLCtx, QuicTlsClientHelloCallback, NULL);
+#ifdef QUIC_WIRESHARK_SUPPORT
+        SSL_CTX_set_keylog_callback(SecurityConfig->SSLCtx, QuicTlsSslKeyLogCallback);
```

Comment:
We should be able to extract this info ourselves when we create the keys, right?